pubtools/exodus/_hooks/pulp.py

```diff
@@ -0,0 +1,210 @@
+import logging
+import os
+import sys
+import time
+
+import attr
+import requests
+from pubtools.pluggy import hookimpl, pm  # pylint: disable=wrong-import-order
+from requests.packages.urllib3.util.retry import (  # pylint: disable=import-error
+    Retry,
+)
+from six.moves.urllib.parse import urljoin
+
+LOG = logging.getLogger("pubtools-exodus")
+
+# pylint: disable=unused-argument
+
+
+class ExodusPulpHandler:
+    def __init__(self):
+        self._gw_url = os.getenv("EXODUS_GW_URL")
+        self._gw_env = os.getenv("EXODUS_GW_ENV")
+        self._gw_crt = os.getenv("EXODUS_GW_CERT")
+        self._gw_key = os.getenv("EXODUS_GW_KEY")
+
+        self._retries = os.getenv("EXODUS_GW_RETRIES") or 4
```

Comment:
This looks buggy:

- if EXODUS_GW_RETRIES is set, then the expression returns a string
- if it's not set, the expression returns an integer

Is self._retries meant to be a string, integer or either?  Make sure it gets converted to the right type.

In this kind of "use env var if set, otherwise use a default" I find it a good habit to ensure the default is always a string.  This ensures you get the same type whether the var is set or not, meaning you can't forget to convert it.